### PR TITLE
Update blocklistConfig.json with BitTorrent trackers & selected lists from Blocklistproject

### PR DESCRIPTION
**Pull request by shuvashish76**
Host list: https://ngosang.github.io/trackerslist/trackers_all.txt
Source: https://github.com/ngosang/trackerslist by @ngosang 
License: [GPL-2.0](https://github.com/ngosang/trackerslist/blob/master/LICENSE)

---

Selected lists from
Source: https://github.com/blocklistproject/Lists by @blocklistproject
License: [unlicense](https://github.com/blocklistproject/Lists/blob/master/LICENSE)
https://unlicense.org/

### CODE REVIEW

**Comment by shuvashish76** [comment]
For future reference
1. used ngosang's trackers_all version ([link](https://ngosang.github.io/trackerslist/trackers_all.txt))
2. what I excluded from blocklistproject's lists (in this pull request): - Abuse, Fraud, Redirect, Scam, Tracking, Smart TV
3. all blocklistproject lists are **Original** version

File: blocklistConfig.json [hunk 1]
```diff
@@ -1224,13 +1224,132 @@
...
+      "group": "privacy",
+      "subg": "",
+      "url": "https://blocklistproject.github.io/Lists/basic.txt"
+    },
+    {
+      "vname": "Blocklistproject Full",
+      "format": "hosts",
+      "group": "privacy",
+      "subg": "",
+      "url": "https://blocklistproject.github.io/Lists/everything.txt
```
Comment (ignoramous) [inline blocklistConfig.json:1352]: missing `"` at the end of the string.

Comment (shuvashish76) [reply blocklistConfig.json]: changed, thanks

**Comment by shuvashish76** [comment]
```
"vname": "BitTorrent trackers All (ngosang)",
 "format": "hosts",
 "group": "privacy",
 "subg": "",
 "url": "https://ngosang.github.io/trackerslist/trackers_all.txt"
```
changed its tag from "parental control" to "privacy" because from what I understand it block Torrent trackers, not websites.

@ignoramous do let me know if any other blocklist groups/sub-groups are in the wrong tag.

**Comment by ignoramous** [comment]
Thanks a lot. I will review the lists to add them when I get back from travels (Sunday).

**Comment by ignoramous** [comment]
Thx. These lists look good to me, except for the names used in the json. I intend to fix that before I merge in the coming days.

**Comment by ignoramous** [comment]
https://ngosang.github.io/trackerslist/trackers_all.txt isn't in `hosts`, `domains`, or `abp` format. Cannot use them.

**Pull request merged**